Doc/library/multiprocessing.rst

```diff
@@ -485,7 +485,9 @@ The :mod:`multiprocessing` package mostly replicates the API of the
    to ``True`` or ``False``.  If ``None`` (the default), this flag will be
    inherited from the creating process.
 
-   By default, no arguments are passed to *target*.
+   By default, no arguments are passed to *target*. The *args* argument,
+   which defaults to ``()``, can be used to specify a list or  tuple of  the arguments
```

Comment:
nitpick: two spaces in here?
![image](https://user-images.githubusercontent.com/9103416/153526933-6ed56aa3-935a-4810-8168-3300e9be1ef0.png)

Reply:
Thanks. I've remove redundant spaces.